### ISSUE TITLE
cell_color property is lost after adding a new row

### ISSUE THREAD

**Issue by touste** [sjmikler/progress-table#32]
When using the `cell_color` option in `update`, then adding a new row, the color is reset to white (only the last row has the correct colors).

I can force the correct behaviour by using `add_row(color=table._data_rows[-1].COLORS)` but this throws an error if `cell_color` was something like "bold green".

```
from progress_table import ProgressTable

table = ProgressTable()
table.add_column("Value")

for i in range(10):
    table.add_row()
    table.update("Value", i, cell_color="red")
```

![image](https://github.com/user-attachments/assets/3a2b0035-732b-4205-9706-fcb46a19108f)

**Comment by szmikler** [comment]
Hey, thanks for catching this. I plan to try reproducing and fixing this in a few days from now.

**Comment by szmikler** [comment]
This should be fixed in #33 and in the newly released version 2.2.5.

Now the cell_color has the priority and will override the general row and column settings. Let me know if that solves it for you.

**Comment by touste** [comment]
I confirm the issue is fixed in the new version, thanks!
